.github/workflows/labeler.yml

```diff
@@ -0,0 +1,12 @@
+name: Compute Labeler
+on:
+- pull_request
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Did you specifically generate this Secret yourself, or is it a standard one already in this repo?

Reply:
this is Github Actions standard generation that came with the template